### ISSUE TITLE
Use jupyter_console for evxr_repl

### ISSUE THREAD

**Issue by Tadaboody** [google/evcxr#139]
Seeing how this project already implements a jupyter kernel, why not use [jupyter_console](https://github.com/jupyter/jupyter_console) It'll give you all that good IPython feel which I think is better than what is currently supported

**Comment by bjorn3** [comment]
I requires jupyter to be installed. In addition IMHO running jupyter in the background is a lot of overhead when all you want is a simple repl session.

**Comment by Tadaboody** [comment]
Fair point, I'd add it as a guide for people who don't know and are already installing jupyter
It's just that the line 
> I'd definitely recommend checking out the Jupyter kernel. It's pretty much a REPL experience, but in a web browser.
Seemed kinda off when `jupyter_console` exists.

**Comment by Tadaboody** [comment]
So I tried running and seems like the unimplemented messages are pretty bothersome :/
```fish
❯ jupyter-console --kernel rust
Got unrecognized message type on shell channel: history_request
Jupyter console 6.2.0

EvCxR 0.5.3 - Evaluation Context for Rust
In [1]: SomeGot unrecognized message type on shell channel: complete_request # Pressed `Tab`
In [1]: Some
```  
(The jupyter lab version prints the same messages in the terminal window you launched from)

I saw that `excvr` does implement some completions, would you accept a pull request that implements the missing messages using them?

**Comment by thomcc** [comment]
Strongly would prefer not having to install jupyter and run a server in the background in order to have evcxr_repl work.

**Comment by davidlattimore** [comment]
Definitely happy to accept pull requests that make jupyter_console work better. Thanks!
